src/Makevars.win

```diff
@@ -0,0 +1,12 @@
+PKG_CPPFLAGS = -DARMA_64BIT_WORD=1 -I.
+PKG_CXXFLAGS = $(SHLIB_OPENMP_CXXFLAGS)
+CXX = /usr/local/opt/llvm/bin/clang++
+CXX11 = clang++
```

Comment:
Same here.

Reply:
Did my most recent change fix the issue? However the issue might persist for MacOS/Linux since I only changed the Makevars.win.